bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -1256,6 +1242,48 @@ public Problem check(Config config, int step, ObservationElements elems, Object
         }
     }
 
+    /**
+     * ERROR: If GMOS-S is used with E2V after semester 2015B.
+     * This can happen when PIs copy parts of old programs into new ones. PIs will not be able to change the detector
+     * settings themselves and will have to have a staff member help them. This is not ideal, but since this is only
```

Comment:
I asked Andy the same question, he prefers to keep this setting disabled for PIs so that they can  not (accidentally) change it.